examples/miller_composite.py

```diff
@@ -0,0 +1,331 @@
+    dy[j+1, :] = dy[j, :]
+
+    for i in range(longitude.shape[1]-1):
+        for j in range(latitude.shape[0]):
+            _, _, dx[j, i] = g.inv(longitude[j, i], latitude[j, i],
+                                   longitude[j, i+1], latitude[j, i+1])
+    dx[:, i+1] = dx[:, i]
+
+    xdiff_sign = np.sign(longitude[0, 1]-longitude[0, 0])
+    ydiff_sign = np.sign(latitude[1, 0]-latitude[0, 0])
+    return xdiff_sign*dx*units.meter, ydiff_sign*dy*units.meter
+
+########################################
+# **Prepare Variables for Plotting**
+#
+# With the data queried and subset, we will make any needed calculations in preparation for
+# plotting.
+#
+# The following fields should be plotted:
+#   500-hPa cyclonic vorticity advection
+#
+#   Surface-based Lifted Index
+#
+#   The axis of the 300-hPa, 500-hPa, and 850-hPa jets
+#
+#   Surface dewpoint
+#
+#   700-hPa dewpoint depression
+#
+#   12-hr surface pressure falls and 500-hPa height changes
+
+
+# 500 hPa CVA
+dx, dy = calc_dx_dy(lon, lat)
+vort_adv_500 = mcalc.advection(avor_500, [v_500.to('m/s'), u_500.to('m/s')], (dy, dx),
+                               dim_order='yx') * 1e9
+vort_adv_500_smooth = gaussian_filter(vort_adv_500, 4)
+
+####################################
+# For the jet axes, we will calculate the windspeed at each level, and plot the highest values
+wspd_300 = gaussian_filter(mcalc.get_wind_speed(u_300, v_300), 5)
+wspd_500 = gaussian_filter(mcalc.get_wind_speed(u_500, v_500), 5)
+wspd_850 = gaussian_filter(mcalc.get_wind_speed(u_850, v_850), 5)
+
+#################################
+# 850-hPa dewpoint will be calculated from RH and temperature
+Td_850 = mcalc.dewpoint_rh(tmp_850, rh_850 / 100.)
+
+################################
+# 700-hPa dewpoint depression will be calculated from temperature and RH
+Td_dep_700 = tmp_700 - mcalc.dewpoint_rh(tmp_700, rh_700 / 100.)
+
+######################################
+# 12-hr surface pressure falls and 500-hPa height changes
+pmsl_change = pmsl - pmsl_00z
+hgt_500_change = hgt_500 - hgt_500_00z
+
+######################################
+# To plot the jet axes, we will mask the wind fields below the upper 1/3 of windspeed.
+
+mask_500 = ma.masked_less_equal(wspd_500, 0.66 * np.max(wspd_500)).mask
+u_500[mask_500] = np.nan
+v_500[mask_500] = np.nan
+
+# 300 hPa
+mask_300 = ma.masked_less_equal(wspd_300, 0.66 * np.max(wspd_300)).mask
+u_300[mask_300] = np.nan
+v_300[mask_300] = np.nan
+
+# 850 hPa
+mask_850 = ma.masked_less_equal(wspd_850, 0.66 * np.max(wspd_850)).mask
+u_850[mask_850] = np.nan
+v_850[mask_850] = np.nan
+
+################################
+# **Create the Plot**
+#
+# With the data now ready, we will create the plot
+
+
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+tlatlons = crs.transform_points(ccrs.PlateCarree(), lon, lat)
```

Comment:
How about passing `transform=ccrs.PlateCarree()` to `contour`, etc., and avoid the manual transform?